### PR TITLE
Adding my name to the participants.csv file

### PR DESCRIPTION
**Pull request by Anais-A**
Test to see if I did everything alright

### CODE REVIEW

**Comment by zaccharieramzi** [comment]
Here you have conflicting file, because you didn't pull the upstream branch which has been modified by 2 prior PRs (#2  and #4 ).

What you need to do is the following:
- Locally, go on your master branch (`git checkout master`).
- Pull the master of the original repo (`git pull upstream master`).
- Go back to your branch (`git checkout mybranch`). Btw, don't forget to have a better branch name next time.
- Now you have to merge the master branch in your feature branch (`git merge master`).
- You will surely have a conflict to handle because the prior PRs also modified the `participants.csv` file. Open your editor and solve the conflict. Btw, take inspiration from the other PRs (#2 and #4 ) to see how to add your name to the `participants.csv` file (that is on a new line, because it's a csv file).
- Once you solved the conflict, stage the file (`git add participants.csv`) and commit it (`git commit -m'solved the merge conflict and corrected the participants file'`).
- You can now push to your distant repo, which will update both your branches on GitHub (`git push`).

Tell me if you have any issues.

File: participants.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Zaccharie Ramzi,
 Guillaume Daval-Frérot,
 Mathilde Ripart,
-
+Anaïs Artiges,
```
Comment (zaccharieramzi) [inline participants.csv:4]: don't remove the whiteline

**Review by zaccharieramzi — CHANGES_REQUESTED** [review]
For future PRs try to keep the changes as minimal as possible, this involves for example not removing or adding whitelines when not necessary.

File: participants.csv (hunk 1 above)
Comment (Anais-A) [reply participants.csv:4]: I think it is okay now!

Comment (zaccharieramzi) [reply participants.csv:4]: no you added your name after the whiteline. The whiteline should be the last line of the file

**Review by zaccharieramzi — APPROVED** [review]
LGTM

**Pull request merged**